packages/blocks/src/root-block/widgets/ai-panel/ai-panel.ts

```diff
@@ -129,7 +134,6 @@ export class AffineAIPanelWidget extends WidgetElement {
     assertExists(text);
     assertExists(this.config.generateAnswer);
 
-    this._resetAbortController();
```

Comment:
Why we delete them (L132 and L147)

Reply:
Example folder has already been configured ignored in the eslint config file but not in lint-staged. So eslint will still check the folder when you try to commit change but it will always fail because of some plugins' missing.

Reply:
Because this will disable the `autoUpdate`, and the AiPanel would then stop moving along with the reference element.